src/Altinn.Platform/Altinn.Platform.Events/Tests/Mocks/EventsPostgresRepositoryMock.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+
+namespace Altinn.Platform.Events.Tests.Mocks
+{
+    /// <summary>
+    /// Class that mocks storing and retrieving documents from postgres DB.
+    /// </summary>
+    public class EventsPostgresRepositoryMock : IEventsPostgresRepository
+    {
+        /// <inheritdoc/>
+        public Task<string> Create(CloudEvent cloudEvent)
+        {
+            return Task.FromResult(string.IsNullOrEmpty(cloudEvent.Id) ? Guid.NewGuid().ToString() : cloudEvent.Id);
```

Comment:
hmm, not sure if returning a new guid here would mask errors in the code. 
Maybe just return an empty string if id is not defined in the cloud event?

Reply:
Maybe drop the check? The Id should never be empty.

Reply:
Done